cpp/arcticdb/python/python_module.cpp

```diff
@@ -25,10 +25,12 @@
 #include <arcticdb/util/global_lifetimes.hpp>
 #include <arcticdb/util/configs_map.hpp>
 #include <arcticdb/util/error_code.hpp>
+#include <arcticdb/python/python_handlers.hpp>
 
 #include <pybind11/pybind11.h>
 #include <folly/system/ThreadName.h>
 #include <folly/portability/PThread.h>
+#include "util/type_handler.hpp"
```

Comment:
For consistency, should this be?
```suggestion
#include <arcticdb/util/type_handler.hpp>
#include <arcticdb/python/python_handlers.hpp>

#include <pybind11/pybind11.h>
#include <folly/system/ThreadName.h>
#include <folly/portability/PThread.h>
```